### PR TITLE
improved console view (possibly also text view)

### LINKED ISSUE #20: Fix output formatting
The final printouts to the console (the table) are badly formatted. I just changed it to comma separation for now, but better to use some printf magic to align the columns and use tabs.

### PR DESCRIPTION
**Pull request by dekespo**
An ugly code that helps the columns adapt according to its attribute (title). Try to give big numbers in different columns and rows, you will see what it exactly does.

It also shows time in rounded to 3 decimal places (hence the change in the function `save_to_html`)

Fixes #20

### CODE REVIEW

File: benchmarker.py [hunk 1]
```diff
@@ -102,7 +115,7 @@ def save_to_html(results):
 	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
 	html += "<tr>" + heading + "</tr>"
 	for i in range(len(languages)):
-		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
+		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str("{:.3f}".format(round(next, 3))) + " s" + "</td>" for next in results[i]])
```
Comment (naslundx) [inline benchmarker.py]: `round(next, 3)` means to round `next` to 3 decimal places, so why the change?

**Comment by naslundx** [comment]
Nice job.
I think a quick fix for `save_to_file` now would be something like

``` python
out_pipe = sys.stdout
sys.stdout = open('w', 'results.txt')
print_to_console(results)
sys.stdout = out_pipe
```

maybe you can try that too?

File: benchmarker.py (hunk 1 above)
Comment (dekespo) [inline benchmarker.py]: Sometimes, it finishes exactly like, for example, 0.06 so instead of printing out 0.060 s, it prints out 0.06 s .It's not big deal, but just aligning everything :) , what do you think?

Comment (naslundx) [inline benchmarker.py]: I see! Then you can remove the round() call?

**Comment by dekespo** [comment]
New version has been updated.

File: benchmarker.py [hunk 2]
```diff
@@ -102,7 +118,7 @@ def save_to_html(results):
 	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
 	html += "<tr>" + heading + "</tr>"
 	for i in range(len(languages)):
-		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
+		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + "{:.3f}".format(next, 3) + " s" + "</td>" for next in results[i]])
```
Comment (naslundx) [inline benchmarker.py]: `format(next, 3)`?

Comment (dekespo) [inline benchmarker.py]: yes, you're right. No need to give precision inside `format` as it is `{:.3f}` already.

**Comment by naslundx** [comment]
:+1:
